docs/sources/developer/architecture/initialization.md

```diff
@@ -0,0 +1,85 @@
+# Initialization process
+
+The initialization process is pretty complex as it has to deal with a lot of components. Below is a step by step
+description of how it is initialized under the hood.
+
+Please refer to the components documentation for more details about each component.
+
+## Core package
+
+### 1. Unpatched Console
+
+The first component that is initialized is the unpatched console. This is needed because it is needed by the internal
```

Comment:
"The first component that is initialized is the unpatched console, because it is required by the internal logger component."?